tests/unit/backend/wmg/test_expression_rollup.py

```diff
@@ -1,49 +1,42 @@
 import unittest
 import numpy as np
-from backend.wmg.pipeline.summary_cubes.rollup import rollup_across_cell_type_descendants
+import pandas as pd
+from backend.wmg.api.rollup import rollup_across_cell_type_descendants
```

Comment:
either this test file should be moved to tests/unit/backend/wmg/api or the module under test should be moved to tests/unit/backend/wmg